lib/jekyll/commands/new.rb

```diff
@@ -121,23 +121,23 @@ def scaffold_path
         # unless the user opts to generate a blank blog or skip 'bundle install'.
 
         def after_install(path, options = {})
-          Jekyll.logger.info "New jekyll site installed in #{path.cyan}."
-          Jekyll.logger.info "Bundle install skipped." if options["skip-bundle"]
-
           unless options["blank"] || options["skip-bundle"]
             bundle_install path
           end
+
+          Jekyll.logger.info "New jekyll site installed in #{path.cyan}."
+          Jekyll.logger.info "Bundle install skipped." if options["skip-bundle"]
         end
```

Comment:
Doesn't it seem like we should actually install Jekyll before we claim that Jekyll was installed?

We _could_ add an additional message before `bundle_install` that the directory was setup, but is it worth it?